extensions/rich_text_components/README.md

```diff
@@ -0,0 +1,6 @@
+Steps to successfully add new Rich Text Component:
+1. Create folder in _extensions/rich_text_components_ with all the files
+2. Create new record in JSON in _asseets/rich_text_components_specs.js_
+3. Add script-imports into _core/templates/dev/head/components/rich_text_components.html_
+
+[More in Wiki](https://github.com/oppia/oppia/wiki/Rich-Text-Editor-%28RTE%29-Overview)
```

Comment:
I suggest linking more specifically, to this: https://github.com/oppia/oppia/wiki/Rich-Text-Editor-%28RTE%29-Overview#rich-text-components

Please also update the wiki page. Thanks!